ccl/workloadccl: TestImportFixture failed

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5121174&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5121174&tab=artifacts#/) on master @ [98bdf3241028c9b1bdff429fb455e61870adc9d0](https://github.com/cockroachdb/cockroach/commits/98bdf3241028c9b1bdff429fb455e61870adc9d0):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:71
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:77
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (46.97s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-15364

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5134776&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5134776&tab=artifacts#/) on master @ [60d0f95ba057dab8d4f50d2903c504fe275d061a](https://github.com/cockroachdb/cockroach/commits/60d0f95ba057dab8d4f50d2903c504fe275d061a):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:71
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:77
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (47.09s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5139894&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5139894&tab=artifacts#/) on master @ [1a91d7cb7b93dfef5dcaf872125875cefa3e0190](https://github.com/cockroachdb/cockroach/commits/1a91d7cb7b93dfef5dcaf872125875cefa3e0190):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:71
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:77
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (47.10s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5149274&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5149274&tab=artifacts#/) on master @ [bc1ee7c7c276984fce8ff5ba4fcfcdff335dde50](https://github.com/cockroachdb/cockroach/commits/bc1ee7c7c276984fce8ff5ba4fcfcdff335dde50):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:71
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:77
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (46.94s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5153049&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5153049&tab=artifacts#/) on master @ [9a179ea1aa8d6723fb12a988f2212ac8493e5dfc](https://github.com/cockroachdb/cockroach/commits/9a179ea1aa8d6723fb12a988f2212ac8493e5dfc):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:71
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:77
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (47.01s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5164418&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5164418&tab=artifacts#/) on master @ [bc5f5b7159ed2b7dbffa94bb2a590fdddb642b9b](https://github.com/cockroachdb/cockroach/commits/bc5f5b7159ed2b7dbffa94bb2a590fdddb642b9b):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:71
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:77
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (46.94s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5190637&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5190637&tab=artifacts#/) on master @ [a35b848edb3f5562199b0f56bc27a6c576073d5d](https://github.com/cockroachdb/cockroach/commits/a35b848edb3f5562199b0f56bc27a6c576073d5d):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:71
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:77
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (47.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/BUILDTYPE_ID-not-found-in-env/5214105?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/BUILDTYPE_ID-not-found-in-env/5214105?buildTab=artifacts#/) on master @ [92947c29c55ff909f50a5e625811d34a1bbe71f7](https://github.com/cockroachdb/cockroach/commits/92947c29c55ff909f50a5e625811d34a1bbe71f7):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:71
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:77
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (47.19s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5224311?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5224311?buildTab=artifacts#/) on master @ [7dd1c51f6b5802e32bafd82e46747f349836592f](https://github.com/cockroachdb/cockroach/commits/7dd1c51f6b5802e32bafd82e46747f349836592f):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:71
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:77
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (47.54s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5251182?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5251182?buildTab=artifacts#/) on master @ [48e48db89eb3cd6bc38f3631364c516181811280](https://github.com/cockroachdb/cockroach/commits/48e48db89eb3cd6bc38f3631364c516181811280):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (47.13s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5291992?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5291992?buildTab=artifacts#/) on master @ [d9fe5f67c75b1b59fc297bf4509a139c640b835b](https://github.com/cockroachdb/cockroach/commits/d9fe5f67c75b1b59fc297bf4509a139c640b835b):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (47.11s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5300411?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5300411?buildTab=artifacts#/) on master @ [1e2cc61b58dc14386bb68dca59814874648931c2](https://github.com/cockroachdb/cockroach/commits/1e2cc61b58dc14386bb68dca59814874648931c2):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (47.18s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5306346?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5306346?buildTab=artifacts#/) on master @ [e6815947a050e32f21e983aa30dc74ab2a247af3](https://github.com/cockroachdb/cockroach/commits/e6815947a050e32f21e983aa30dc74ab2a247af3):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (47.09s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5312586?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5312586?buildTab=artifacts#/) on master @ [e6815947a050e32f21e983aa30dc74ab2a247af3](https://github.com/cockroachdb/cockroach/commits/e6815947a050e32f21e983aa30dc74ab2a247af3):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (46.91s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/workloadccl.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5323577?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5323577?buildTab=artifacts#/) on master @ [258551336a7643f8db13df8ff7ab65d678eae7eb](https://github.com/cockroachdb/cockroach/commits/258551336a7643f8db13df8ff7ab65d678eae7eb):


```
=== RUN   TestImportFixture
    fixture_test.go:204: 
        	Error Trace:	sql_runner.go:86
        	            				sql_runner.go:251
        	            				fixture_test.go:204
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:254
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:207
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:86
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:251
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test.TestImportFixture
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl_test/pkg/ccl/workloadccl/fixture_test.go:204
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | got:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestImportFixture
--- FAIL: TestImportFixture (46.92s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/workloadccl/fixture_test.go
+++ b/pkg/ccl/workloadccl/fixture_test.go
@@ -166,8 +166,6 @@ func TestFixture(t *testing.T) {
 func TestImportFixture(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 
-	skip.WithIssue(t, 81082, "flaky")
-
 	ctx := context.Background()
 
 	defer func(oldRefreshInterval, oldAsOf time.Duration) {
@@ -193,6 +191,10 @@ func TestImportFixture(t *testing.T) {
 	if err := gen.Flags().Parse([]string{"--" + flag}); err != nil {
 		t.Fatalf(`%+v`, err)
 	}
+	// Wait for the `ensureAllTables` unconditional auto stats to run before
+	// starting the test, so we don't hit a timing window where 2 sets of auto
+	// stats are collected.
+	time.Sleep(2 * time.Second)
 
 	const filesPerNode = 1
 

